x/bank/simulation/operations.go

```diff
@@ -80,6 +80,35 @@ func SimulateMsgSend(ak types.AccountKeeper, bk keeper.Keeper) simtypes.Operatio
 	}
 }
 
+// SimulateMsgSendToModuleAccount tests and runs a single msg send where both
+// accounts already exist.
+func SimulateMsgSendToModuleAccount(ak types.AccountKeeper, bk keeper.Keeper) simtypes.Operation {
+	return func(
+		r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context,
+		accs []simtypes.Account, chainID string,
+	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
+		simAccount := accs[0]
+		toSimAcc := accs[1]
```

Comment:
How about using `fromSimAcc`, `toSimAcc` or `from`, `to`? I'll pick `from` and `to`.